database-sample-data.json

```diff
@@ -70,12 +70,14 @@
       "round_1": {
         "delegation": "delegation_id_1",
         "markedAsSent": true,
+        "markedQuestionAsSent": true,
```

Comment:
I would rename to `markedLandsraadQuestionsAsSent`, that's more clear

Reply:
fixed.